nemo_skills/dataset/minif2f/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo_skills.evaluation.metrics import MathMetrics
+
+# Default evaluation and generation settings for the minif2f dataset
+PROMPT_CONFIG = 'generic/lean4'
+DATASET_GROUP = 'lean4'
+METRICS_CLASS = MathMetrics
```

Comment:
Let's create a new metrics class here. We don't have support for majority voting, llm judgement and want to call metrics differently. It can reuse the same is_correct key, but let's remove all un-neded logic and name the metrics appropriately. If there is some more precise metric reporting that we can do (e.g. counting timeouts or syntax errors separately) that's good to add as well